libethcore/EVMSchedule.h

```diff
@@ -29,6 +29,7 @@ struct EVMSchedule
 {
     EVMSchedule(): tierStepGas(std::array<unsigned, 8>{{0, 2, 3, 5, 8, 10, 20, 0}}) {}
     EVMSchedule(bool _efcd, bool _hdc, unsigned const& _txCreateGas): exceptionalFailedCodeDeposit(_efcd), haveDelegateCall(_hdc), tierStepGas(std::array<unsigned, 8>{{0, 2, 3, 5, 8, 10, 20, 0}}), txCreateGas(_txCreateGas) {}
+    unsigned version = 0;
```

Comment:
Is this the best place to store the version? EVMSchedule appears to only contain opcode / gas cost info, is there another place where we store other VM configuration information?

Reply:
There's also at least block reward value, and some other things. I think it's an ok place, I don't think there is any other currently.